BrainPortal/app/views/portal/welcome.html.erb

```diff
@@ -167,14 +167,15 @@
       <strong>Your current time:</strong> <%= to_localtime(Time.now, :datetime) %><BR>
     </p>
     <p>
-       <strong>Projects you belong to:</strong>  <%= array_to_table(@groups, :table_class => 'simple', :cols => 4) { |g,r,c| link_to_group_if_accessible(g) } %>
+    <p>
+      <strong> The complete list of curated datasets and all <%= @tool_names.count %> tools on this portal:</strong> <%= link_to 'here.', available_path %>
```

Comment:
Yuk, you get

```
The complete list of curated datasets and all 1234 tools on this portal: here.
```

Also, why not print this on the same line as "Tools available to you:" ?

```
Tools available to you (34 of 845, [link]full list of all tools[/link])
```

Reply:
A good web design convention is to never create links with just the value "here" as the anchor. Use a full descriptive sentence about what the user will get when they click the link.

Reply:
Totally agree. I tried preserve the original text and colon based. I guess I should not goof around and cut and paste the original sentence, perhaps even without ' is here'.

Well, what you proposed is not so bad, though I do not like particularly 

a)  long overloaded sentence; 
b) long text in brackets; 
c) here is still there, I'd rather drop it, otherwise user might got confused, to me here is rather page not necessarily the portal as used by all the user ;  
d) mentioning of curated datasets are disappeared

Not sure maybe 

`Tools available to you ( 34  of   845 [link] of all the configured tools[/link]):`

`Tools available to you ([ link to versions ] 34 ] of  [link] 845 of all the supported [/link]):`

I'd keep then old link below for sake of dataset/